packages/drivers/odsp-driver/src/odspDriverUrlResolver.ts

```diff
@@ -156,23 +155,14 @@ export class OdspDriverUrlResolver implements IUrlResolver {
     public async getAbsoluteUrl(
         resolvedUrl: IResolvedUrl,
         relativeUrl: string,
-        packageInfoSource?: IContainerPackageInfo | IFluidCodeDetails,
+        packageInfoSource?: IContainerPackageInfo,
     ): Promise<string> {
         let dataStorePath = relativeUrl;
         if (dataStorePath.startsWith("/")) {
             dataStorePath = dataStorePath.substr(1);
         }
         const odspResolvedUrl = getOdspResolvedUrl(resolvedUrl);
-
-        // back-compat: IFluidCodeDetails usage to be removed in 0.58.0
-        let containerPackageName;
-        if (packageInfoSource && "name" in packageInfoSource) {
```

Comment:
I'm not super familiar with how/if our customers are calling this today.  If there's any risk that someone might still make a call using `IFluidCodeDetails`, perhaps consider adding an assert here to catch it (same for `odspDriverUrlResolverForShareLink.ts`).

At minimum it looks like an older version of the `Container` could potentially call into this newer driver and blow up -- I think it might not be safe to remove this logic for that reason.